sources/models/src/aws-k8s-1.15/defaults.d/50-aws-k8s.toml

```diff
@@ -58,6 +63,8 @@ affected-services = ["kubernetes", "containerd"]
 cluster-domain = "cluster.local"
 standalone-mode = false
 authentication-mode = "aws"
+kube-reserved.ephemeral-storage = "1Gi"
+cpu-manager-policy = "none"
```

Comment:
Are these values that we were already implicitly defaulting to?

Reply:
No. I just add those values for those new settings `kube-reserved` and `cpu-manager-policy`